pkg/utils/utils.go

```diff
@@ -180,11 +181,15 @@ func GetFileContents(filePath string) []byte {
 }
 
 func GetShareDir() string {
-	shareDir := os.Getenv("LOGGING_SHARE_DIR")
-	if shareDir == "" {
-		return "/usr/share/logging"
-	}
-	return shareDir
+	// shareDir is <repo-root>/files - try to find from working dir.
+	const sep = string(os.PathSeparator)
+	const repoRoot = sep + "cluster-logging-operator" + sep
+	wd, _ := os.Getwd()
```

Comment:
@alanconway why suppress errors?

Reply:
The outcome is the same - failure of Getwd() will return "", and we definitely won't find /cluster-logging-operator/ in that.
However you are right that it would be clearer to handle the error, so I'll fix that and put a postit on my screen saying "Shortcuts Make Long Delays"  ;)